core/src/main/java/org/apache/accumulo/core/metadata/schema/MetadataSchema.java

```diff
@@ -433,6 +420,23 @@ public static class HostingColumnFamily {
       public static final String REQUESTED_QUAL = "requested";
       public static final ColumnFQ REQUESTED_COLUMN = new ColumnFQ(NAME, new Text(REQUESTED_QUAL));
     }
+
+    // These can be removed when the corresponding upgrade code is removed
+    public static class Upgrade12to13 {
```

Comment:
Why not move these into the Upgrader itself. When the upgrader gets deleted, then we don't need to remember to delete these too.

Reply:
I started with that approach and then in testing I ran into a problem where the upgrade code could not write to the metadata table because of the metadata table constraint.  So I had to make the constraint code still use these columns and the package it is in can not access the Upgrader (constraint is in server or core while Upgrader is in manager jar).

So I ended up settling on this approach of isolating the columns in the MetadataSchema class.